CAPI/KubeadmControlplane.yaml

```diff
@@ -24,3 +24,27 @@ spec:
           labels:
             +(cluster.x-k8s.io/watch-filter): "{{ releases[?metadata.name == 'v{{ clusters[?metadata.name == '{{request.object.metadata.labels.\"cluster.x-k8s.io/cluster-name\"}}'].metadata.labels.\"release.giantswarm.io/version\"  | [0] }}'].spec.components[] | [?name == 'cluster-api-control-plane'].version | [0]}}"
             cluster.x-k8s.io/watch-filter: "{{ releases[?metadata.name == 'v{{ clusters[?metadata.name == '{{request.object.metadata.labels.\"cluster.x-k8s.io/cluster-name\"}}'].metadata.labels.\"release.giantswarm.io/version\"  | [0] }}'].spec.components[] | [?name == 'cluster-api-control-plane'].version | [0]}}"
+  - name: kubeadmcontrolplane-metricendpoints
+    match:
+      resources:
+        kind: 
+        - controlplane.cluster.x-k8s.io/v1alpha3/KubeadmControlPlane
+    context:
+    - name: clusters
+      apiCall:
+        urlPath: "/apis/cluster.x-k8s.io/v1alpha3/namespaces/{{ request.object.metadata.namespace }}/clusters"
+        jmesPath: "items[*]"
+    mutate:
```

Comment:
I think you can just put this in the same mutate as above (I think)

Reply:
Probably, but naming :D

Reply:
I prefer it to not become too convoluted as well

Reply:
I tried to simplify it, let's see if it works first :D